src/main/java/edition/academy/seventh/repository/BookRepository.java

```diff
@@ -7,6 +7,9 @@
 
```

Comment:
Ten plik jest już na develop! Dlaczego ląduje on 2 raz i to w nie związanym PR? Uaktualnij u siebie gałąź  żeby to nie szło 2 raz.

Reply:
nie wiem co sie podziało, musi tak być, bo inaczej będzie konfllikt